README.md

```diff
@@ -1,8 +1,103 @@
+CS:GO, and TF2.*
 
-Lumper is a .NET based tool developed by the Momentum Mod Team for manipulating BSP files, the Source engine's compiled map format.
-It is aimed at map reviewers and porters, giving them the ability to analyse and tweak BSP files without having to decompile then recompile.
+<h1 style="display: flex; align-items: center; gap: 0.5rem;">
+    <img src="images/Lumper.svg" alt="Lumper Logo" width="32" height="32">
+	<span>Lumper</span>
+</h1>
 
-Potential contributors are welcome to reach out to us in the [Momentum Mod Discord server](https://discord.gg/momentummod).
+Lumper is a program for viewing and manipulating the various "lumps" that constitute Source Engine map files (BSPs).
+
+It is *not* a map authoring tool like Hammer, which operates on map source files (VMFs) that are then
+compiled to BSPs. Lumper is intended for small, destructive tweaks to existing maps. It's most similar to
+[VIDE](http://www.riintouge.com/VIDE/), which is unfortunately closed-source, inactive, and doesn't support more recent
+BSP versions.
+
+## Features
+
+### Entity Editor
+
+View and edit the contents of the entity lump à la EntSpy/VIDE. Includes a "Raw Entities" view for editing the entire
+lump as plaintext.
+
+![Entity Editor](images/entity_editor.png)
+
+### Pakfile Editor
+
+View and edit the contents of the pakfile lump with a built-in file tree, text editor, and VTF viewer.
+It supports importing files/directories, exporting to the filesystem, and can automatically update references to paths in
+other lumps when items are moved.
+
+![Pakfile Explorer](images/pakfile_explorer.png)
+
+### Texture Browser
+
+Displays every VTF file packed into the pakfile, like Hammer's texture UI.
+
+![Texture Browser](images/texture_browser.png)
+
+### Jobs
+
+Jobs are a way of automating repetitive tasks, particularly for Momentum map porting. Multiple jobs can be ordered
+sequentially, and export/imported as JSON "workflow" files. The current jobs we support are:
+
+- __Stripper__ - Apply [Stripper:Source](https://www.bailopan.net/stripper/) configs to a BSP. What is usually done at
+  runtime on a server via SourceMod can be *permanently* applied to the BSP file!
+- __Replace Textures__ - Update the texdata lumps to refer to different material files. Supports regex!
+- __Run External Tool__ - Runs a given executable on the loaded BSP.
+
+![Jobs](images/jobs.png)
+
+### Compressed Exporting
+
+Lumper can load and save LZMA compressed BSPs, including the pakfile, producing the same result
+as [`bspzip`](https://developer.valvesoftware.com/wiki/BSPZIP).
+
+### JSON Diffs
+
+Lumper can serialize the contents of the BSP to JSON files, helpful for analysing a BSP in a human-readable format and
+diffing BSP versions in Git.
+
+## Download
+
+See the [Releases](https://github.com/momentum-mod/lumper/releases) tab on the right! This program is very new, and we

```

Comment:
"new" .. I wonder who is going to change this and when. :) I would get rid of the second sentence here. People don't expect everything they download from github to auto-update, do they?

Reply:
I agree about the dotnet version in the readme, but in this case I think it's fine. We're likely to change release stuff in the future and I don't think it's at all unreasonable to update the readme when the program is more stable.